### ISSUE TITLE
Improve building instructions

### ISSUE THREAD

**Issue by kattjevfel** [GTAmodding/re3#799]
Currently when building re3 you're forced to look at two locations for instructions and fill the gaps where needed and then just cross your fingers it works.

For example it tells you to run `./premake5Linux --with-librw gmake2` but since you're installing software anyway, you'd be better off just using a packaged version of premake instead of an unknown (and a bit sketchy) provided binary. and what does `--with-librw` do? The README.md mentions being able to set `LIBRW` but that seems to only control the source location of librw instead of compiled version (please document!).

Back to the "Building on linux" page, it tells you to run make with "for example" one config, but it doesn't tell you about any other examples, or how they work, can I use a system-wide librw?
It then tells the user to copy the elf file, which is done automatically anyway, so that's misleading.

Furthermore, CMake support was recently added, but none of this "config" stuff works anymore, and there's literally no documentation on it, so my hopes of escaping premake-hell is looking dark so far.

*Please* add some better documentation for how to use the different build options!

**Comment by hazelnot** [comment]
>It then tells the user to copy the elf file, which is done automatically anyway, so that's misleading.

Honestly, setting an environmental variable just for the script to know where to put the executable is really weird as well, I'd rather just copy the executable than do this, plus #746 exists which would fix all of this nonsense, but it's not merged yet. I know my opinion doesn't really matter that much since I'm not a programmer, but the way this project is handled is just bizzare as hell.

**Comment by kattjevfel** [comment]
> > It then tells the user to copy the elf file, which is done automatically anyway, so that's misleading.
> 
> Honestly, setting an environmental variable just for the script to know where to put the executable is really weird as well, I'd rather just copy the executable than do this, plus #746 exists which would fix all of this nonsense, but it's not merged yet. I know my opinion doesn't really matter that much since I'm not a programmer, but the way this project is handled is just bizzare as hell.

Yeah I was expecting it to only need that for sourcing original game files, but it puts the executable there as well. Would love for an option to be able to manually specify the location (so it can for example be placed in /usr/share for contents and /usr/bin for the exec), and so on.
